lib/pusher/webhook.rb

```diff
@@ -92,14 +92,24 @@ def data
       end
     end
 
+    # Time constant string comparison
+    #
+    def secure_compare(a, b)
+      return false unless a.bytesize == b.bytesize
+      l = a.unpack "C#{a.bytesize}"
+      res = 0
+      b.each_byte { |byte| res |= byte ^ l.shift }
+      res == 0
+    end
```

Comment:
We may have to credit the original implementation https://github.com/rails/rails/blob/c8c660002f4b0e9606de96325f20b95248b6ff2d/activesupport/lib/active_support/security_utils.rb#L9